src/ui/ErrorReport/ErrorReport.ts

```diff
@@ -305,7 +305,7 @@ export class ErrorReport extends Component {
     this.container.append(optionsElement.el);
   }
 
-  private buildEndpointErrorElements(helpLink: string = 'http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=254') {
+  private buildEndpointErrorElements(helpLink: string = 'https://docs.coveo.com/en/0/') {
```

Comment:
Well this won't help them much :)

I'd still point to https://docs.coveo.com/en/331/